api/v1/models.go

```diff
@@ -124,7 +124,7 @@ func LayerFromDatabaseModel(db database.Datastore, dbLayer database.Layer, linea
 
 	notes := getNotes(layer.NamespaceName, uncertifiedRHEL)
 
-	if (withFeatures || withVulnerabilities) && (dbLayer.Features != nil || namespaces.IsRHELNamespace(layer.NamespaceName)) {
+	if (withFeatures || withVulnerabilities) && (dbLayer.Features != nil || namespaces.IsRHELNamespace(layer.NamespaceName) || namespaces.IsRHCOSNamespace(layer.NamespaceName)) {
```

Comment:
Why did you change this file? Did you see that the code in Compliance hits these lines?

Reply:
Thank you for spotting this.
I added this in debugging the empty component list.
I have verified that package detection works without this change, so I have rolled it back.

Reply:
You've reverted one change, but I still see two changed spots in `api/v1/models.go`.